include/mbedtls/md.h

```diff
@@ -130,6 +139,7 @@ const int *mbedtls_md_list(void);
  * \return          NULL if the associated message-digest information is not found.
  */
 const mbedtls_md_info_t *mbedtls_md_info_from_string(const char *md_name);
+#endif /* MBEDTLS_MD_C */
```

Comment:
Suggestion: Maybe we could group functions that need `MBEDTLS_MD_C` in one place?